jwst/jump/jump.py

```diff
@@ -11,7 +11,11 @@
 def run_detect_jumps(input_model, gain_model, readnoise_model,
                      rejection_thresh, three_grp_thresh, four_grp_thresh,
                      max_cores, max_jump_to_flag_neighbors,
-                     min_jump_to_flag_neighbors, flag_4_neighbors):
+                     min_jump_to_flag_neighbors, flag_4_neighbors,
+                     after_jump_flag_dn1=0.0,
```

Comment:
change param name to match stcal

Reply:
I think the confusion here is that it's setup intentionally for the user to give these argument values in units of DN, because that's what they see in the ramp data files, but because the lower-level routines in stcal work on the data after they've been (temporarily) converted to units of electrons, these argument values also get converted down in those routines and hence the name change. At the top levels they're DN, but in the lower level stcal functions they're electrons. So this apparent inconsistency in the naming is OK (IMHO).

Reply:
ok i see this now - i will remove my comments about this.

the input to detect_jumps in stcal is DN, its converted there to e-, and the input to find_crs in then electrons. i would vote for doing the conversion in side find_crs to keep all the units the same. i just see this possibly getting confusing when setting up tests that run the full step vs. calling find_crs directly.

Reply:
I did the conversion at this point as it is where the data is converted to electrons.  Seemed like the consistent place to do it.